test/python/README.md

```diff
@@ -8,11 +8,11 @@ To run the tests for python binding of **rticonnextdds_connector**:
 
 2. To execute all the tests, issue the following command from the base directory: 
   
-   ``py.test ./test/python``
+   ``pytest ./test/python``
```

Comment:
Isn't it py.test?

Reply:
I can use py.test or python -m pytest, but not just pytest.

Reply:
py.test wasn't working for me, only pytest. I was reading up on it and this answer: https://stackoverflow.com/questions/39495429/py-test-vs-pytest-command seemed to suggest pytest was the correct one to use. I wonder what is different in our installations?